src/SecondOrderSeawaterPolynomials.jl

```diff
@@ -136,7 +147,7 @@ RoquetEquationOfState(FT::DataType, coefficient_set=:SecondOrder; reference_dens
     BoussinesqEquationOfState(RoquetSeawaterPolynomial(FT, coefficient_set), reference_density)
 
 RoquetEquationOfState(coefficient_set=:SecondOrder; reference_density=1024.6) =
-    RoquetEquationOfState(Float64, coefficient_set, reference_density=1024.6)
+    RoquetEquationOfState(Float64, coefficient_set, reference_density)
```

Comment:
```suggestion
    RoquetEquationOfState(Float64, coefficient_set; reference_density)
```

nice catch!
but to pass a keyword arg just with a variable name you need to add it after `;`.

this seems like a bug since regardless the value of `reference_density` the method would had always pass `1024.6`.

cc @glwagner

Reply:
Definitely a good catch! I'm not aware of anyone who has ever used this. But we'll have to bump minor version and all that

Reply:
yeah